packages/server/__snapshots__/0_reporters_spec.js

```diff
@@ -12,10 +12,7 @@ Cannot find module '/foo/bar/.projects/e2e/node_modules/module-does-not-exist'
 Require stack:
 - lib/reporter.js
 - lib/project-base.ts
-- lib/project_static.ts
-- lib/modes/run.js
-- lib/modes/run-e2e.js
-- lib/modes/index.js
+- lib/open_project.ts
```

Comment:
Not entirely sure why this snapshot needs changing, I guess using TS changes how the modules are resolved and the stack trace.